.github/workflows/dependency-review.yml

```diff
@@ -23,22 +23,13 @@ jobs:
           # compatible/incompatible licenses addressed here: https://www.apache.org/legal/resolved.html
           # find SPDX identifiers here: https://spdx.org/licenses/
           deny-licenses: MS-LPL, BUSL-1.1, QPL-1.0, Sleepycat, SSPL-1.0, CPOL-1.02, AGPL-3.0, GPL-1.0+, BSD-4-Clause-UC, NPL-1.0, NPL-1.1, JSON
-          allow-dependencies-licenses:
-            # adding an exception for an ambigious license on store2, which has been resolved in
-            # the latest version. It's MIT: https://github.com/nbubna/store/blob/master/LICENSE-MIT
-            - 'pkg:npm/store2@2.14.2'
-            # adding exception for all applitools modules (eyes-cypress and its dependencies),
-            # which has an explicit OSS license approved by ASF
-            # license: https://applitools.com/legal/open-source-terms-of-use/
-            - 'pkg:npm/applitools/core'
-            - 'pkg:npm/applitools/core-base'
-            - 'pkg:npm/applitools/css-tree'
-            - 'pkg:npm/applitools/ec-client'
-            - 'pkg:npm/applitools/eg-socks5-proxy-server'
-            - 'pkg:npm/applitools/eyes'
-            - 'pkg:npm/applitools/eyes-cypress'
-            - 'pkg:npm/applitools/nml-client'
-            - 'pkg:npm/applitools/tunnel-client'
-            - 'pkg:npm/applitools/utils'
-            # Selecting BSD-3-Clause licensing terms for node-forge to ensure compatibility with Apache
-            - 'pkg:npm/node-forge@1.3.1'
+          # pkg:npm/store2@2.14.2
+          #   adding an exception for an ambigious license on store2, which has been resolved in
+          #   the latest version. It's MIT: https://github.com/nbubna/store/blob/master/LICENSE-MIT
+          # pkg:npm/applitools/*
+          #   adding exception for all applitools modules (eyes-cypress and its dependencies),
+          #   which has an explicit OSS license approved by ASF
+          #   license: https://applitools.com/legal/open-source-terms-of-use/
+          # pkg:npm/node-forge@1.3.1
+          #   selecting BSD-3-Clause licensing terms for node-forge to ensure compatibility with Apache
+          allow-dependencies-licenses: pkg:npm/store2@2.14.2, pkg:npm/applitools/core, pkg:npm/applitools/core-base, pkg:npm/applitools/css-tree, pkg:npm/applitools/ec-client, pkg:npm/applitools/eg-socks5-proxy-server, pkg:npm/applitools/eyes, pkg:npm/applitools/eyes-cypress, pkg:npm/applitools/nml-client, pkg:npm/applitools/tunnel-client, pkg:npm/applitools/utils, pkg:npm/node-forge@1.3.1
```

Comment:
Any reason to switch from multiline to single line? This seems less readable and harder to review future changes if only one dependency changes.

Reply:
The GH action only accepts the input arg as single line. Maybe it will get fixed in the future version but for now, it's the only way proceeding